docs/getting-started-objective-c.md

```diff
@@ -28,6 +28,9 @@ Your next step is to download and install the embeddinator on your Mac.
 
 There is no package to download yet. Until one is available you need to build it from our git repository, see the [constributing](Contributing#Objective-C) document for instructions.
 
+Installers can be built by running "make installer" in the objcgen directory.
```

Comment:
that should be mentioned in the Contributing document - not in the getting started :)